src/en/general-development/tips/forking.md

```diff
@@ -0,0 +1,105 @@
+# Forking Tips
+
+A general set of forking tips and agreed upon best practices from fork devs. If you are a fork dev that would like to add something here please open a PR.
+
+It is strongly recommended that you have some experience with SS14 and RT before creating your own fork.
+
+This document only describes the technical considerations of forking. Keep in mind that any good fork will require an administration team and policies.
+
+## Creating Your Repo
+
+You should not create your project's repository with the github fork button. This is because github only permits one fork of a repo per account and forks of your repo will be counted the same as forks of your upstream. Additionally, it makes it impossible to accidentally send pull requests to your upstream.
+
+Instead, do the following:
+- Create blank repo, either on an org or in your user. Do not initialize the repo with an initial commit, leave it blank.
+- git clone the repository of the upstream you wish to fork from (if you already have a local copy, then you can use that instead)
+- `git remote add YourFork github.com/YourOrg/YourRepo` to add your blank repo as a remote
+- git checkout your desired remote branch. If you're basing off Wizard's den, it is strongly recommended that you base your fork on stable.
+    - `git fetch upstream stable` (assuming the remote pointing to the repository you wish to fork from is called upstream)
+    - `git checkout upstream/stable`
+- `git push YourFork master` to push that to your fork as the master branch.
+
+Congratulations, you have successfully started a fork!
+
+To update your fork in the future do as flows:
+- Create a new branch based on the most recent commit of your fork
+- git fetch your desired upstream branch as above
+- Then git pull it.
+
+Continuing with the example of mainline stable, the commands used for the above would be:
+- `git fetch upstream stable`
+- `git pull upstream stable`
+- Fix merge conflicts if needed, and PR your fresh upstream merge branch
+- If you do not want an upstream feature, now is the time to revert the change using `git revert`
+
+Note that any contributors to your fork will need to press the github fork button on your repository instead of your upstream's.
+
```

Comment:
We should add a short section here summarizing the basics of copyright/licensing to help prevent people from falling into common pitfalls. I'll write something up this week unless you'd like to write something up and I can look it over :)
Once we've got that, this will be good to merge!

Reply:
I actually had some something small about the AGPL (most forks are not using it correctly), but opted not to include it because I did not want to start shit due to how much of a hot topic that is in ss14. We could link https://choosealicense.com/ which is written by github and I assume vetted by their lawers.

Reply:
I think one of the main issues with the above site is it encourages using the GPL over the AGPL for a copyleft license which is sound advice for most projects, but will probably not do what they intend for a ss14 fork.